polyfill/lib/plainmonthday.mjs

```diff
@@ -127,7 +127,7 @@ export class PlainMonthDay {
     const fields = ES.ToTemporalMonthDayFields(this, receiverFieldNames);
 
     const inputFieldNames = ES.CalendarFields(calendar, ['year']);
-    const entries = [['year']];
+    const entries = [['year', undefined]];
```

Comment:
I checked, and this is only a bug in the polyfill, the spec text is correct. Thanks for catching it! Would you mind adding a test (feel free to postpone this or add it to #1398)